contracts/0.8.9/WithdrawalNFT.sol

```diff
@@ -0,0 +1,234 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>, OpenZeppelin
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.8.9;
+
+import {IERC721} from "@openzeppelin/contracts-v4.4/token/ERC721/IERC721.sol";
+import {IERC721Receiver} from "@openzeppelin/contracts-v4.4/token/ERC721/IERC721Receiver.sol";
+import {IERC165} from "@openzeppelin/contracts-v4.4/utils/introspection/ERC165.sol";
+
+import {Strings} from "@openzeppelin/contracts-v4.4/utils/Strings.sol";
+import {ERC165} from "@openzeppelin/contracts-v4.4/utils/introspection/ERC165.sol";
+import {EnumerableSet} from "@openzeppelin/contracts-v4.4/utils/structs/EnumerableSet.sol";
+import {Address} from "@openzeppelin/contracts-v4.4/utils/Address.sol";
+
+import {IWstETH, WithdrawalQueue} from "./WithdrawalQueue.sol";
+import {IAccessControlEnumerable, AccessControlEnumerable} from "./utils/access/AccessControlEnumerable.sol";
```

Comment:
`IAccessControlEnumerable` isn't used anywhere
should the `supportsInterface` include it?